README.md

```diff
@@ -158,6 +158,11 @@ markers.on('clusterclick', function (a) {
 #### Bulk adding and removing Markers
 `addLayers` and `removeLayers` are bulk methods for adding and removing markers and should be favoured over the single versions when doing bulk addition/removal of markers. Each takes an array of markers. You can use [dedicated options](#chunked-addlayers) to fine-tune the behaviour of `addLayers`.
 
+These methods extract non-group layer children from Layer Group types, even deeply nested. _However_, be noted that:
+- The input array is modified. If you need to keep it untouched, clone it before passing it (e.g. `newArray = originalArray.slice()`).
```

Comment:
Maybe "The input array is modified if it has LayerGroups inside it?"... Looking back at it this is sorta ugly though, maybe we just take the performance hit?

Could we compare the perf of always cloning the array or scanning and cloning if needed?